examples/mlrun_sparkk8s.ipynb

```diff
@@ -160,7 +160,11 @@
     "\n",
```

Comment:
<p>can this be deleted since you moved it to commands?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/3273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Still needed by the local run.</p>